yt_dlp/extractor/abc.py

```diff
@@ -299,47 +329,60 @@ def tokenize_url(url, token):
                 'hdnea': token,
             })
 
-        for sd in ('1080', '720', 'sd', 'sd-low'):
-            sd_url = try_get(
-                stream, lambda x: x['streams']['hls'][sd], str)
-            if not sd_url:
-                continue
-            formats = self._extract_m3u8_formats(
-                tokenize_url(sd_url, token), video_id, 'mp4',
-                entry_protocol='m3u8_native', m3u8_id='hls', fatal=False)
-            if formats:
-                break
+        formats = []
+        # hls: pre-merged formats
+        # hls-latest: same as hls, but sometimes has separate audio tracks
+        # Note: pre-merged formats in hls-latest are treated as video-only if audio-only tracks
+        # are present, so we extract both types
+        for label in ('hls', 'hls-latest'):
+            for sd_url in traverse_obj(stream, ('streams', label, ('1080', '720', 'sd', 'sd-low'), {str})):
+                fmts = self._extract_m3u8_formats(
+                    tokenize_url(sd_url, token), video_id, 'mp4',
+                    entry_protocol='m3u8_native', m3u8_id=label, fatal=False)
+                if fmts:
+                    formats.extend(fmts)
+                    break
 
         subtitles = {}
-        src_vtt = stream.get('captions', {}).get('src-vtt')
+        src_vtt = traverse_obj(stream, ('captions', 'src-vtt'))
         if src_vtt:
             subtitles['en'] = [{
                 'url': src_vtt,
                 'ext': 'vtt',
             }]
 
-        is_live = video_params.get('livestream') == '1'
+        title = unescapeHTML(traverse_obj(video_params, 'title', 'seriesTitle', 'showTitle'))
 
         return {
             'id': video_id,
             'title': title,
```

Comment:
inline `title` and transform using the `expected_type` param

```suggestion
        return {
            'id': video_id,
            'title': traverse_obj(video_params, 'title', 'seriesTitle', 'showTitle', expected_type=unescapeHTML),
```